.github/workflows/e2e-test.yml

```
@@ -176,38 +190,18 @@ jobs:
       - name: Run tests, upload screenshots & logs
         uses: ./.github/actions/e2e/run-log-tests
 
-  # Run tests against WC Checkout blocks & WC latest
-  # [TODO] Unskip blocks tests after investigating constant failures.
-  # blocks-tests:
-  #   runs-on: ubuntu-latest
-  #   name: WC - latest | blocks - shopper
 
-  #   env:
-  #     E2E_WP_VERSION: 'latest'
-  #     E2E_WC_VERSION: 'latest'
-  #     E2E_GROUP:  'blocks'
-  #     E2E_BRANCH: 'shopper'
-  #     SKIP_WC_SUBSCRIPTIONS_TESTS: 1 #skip installing & running subscriptions tests
-  #     SKIP_WC_ACTION_SCHEDULER_TESTS: 1 #skip installing & running action scheduler tests
-
-  #   steps:
-  #     - name: Checkout WCPay repository
-  #       uses: actions/checkout@v4
-
-  #     - name: Setup E2E environment
-  #       uses: ./.github/actions/e2e/env-setup
-
-  #     - name: Run tests, upload screenshots & logs
-  #       uses: ./.github/actions/e2e/run-log-tests
 
   # Run tests against WP Nightly & WC latest
+  # This section tests cutting-edge WordPress features with latest WC
```

Nitpick: 😬 I don't think WordPress always has state-of-the-art/cutting-edge features for each of its major releases, so using the `new` is quite enough IMO.
```suggestion
  # This section tests new WordPress features with latest WC
```

---

👍 I'm using "new" instead of cutting-edge thanks!